### DISCUSSION TITLE
Long loading times when using GitHub

### DISCUSSION THREAD

**Discussion by unordentlich** [community/community#42515] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hey there!
I am experiencing extremely long loading times when using GitHub on the web, so it sometimes takes up to 3-4 seconds, when navigating to a new directory.

I am not sure what could be causing these delays, but I wanted to ask you if there is anything that can be done to resolve the problem.

I would appreciate your help with this, thank you in advance for your help.

**Comment by jfullstackdev** [comment, marked as answer]
that depends how big the file is 

but in my computer this is an introductory level gaming Windows machine, I never had that problem, even in Jupyter Notebook that they post here, my computer can load them very fast, just check your RAM and those antivirus and firewall 

another one is maybe your connection if it's wireless, better use the cable 

GitHub Extensions installed on your browser can affect GitHub Web too

**Reply by unordentlich** [reply]
Thank you!
